desktop/core/src/desktop/views.py

```diff
@@ -403,7 +417,7 @@ def status_bar(request):
         resp += r.content
       else:
         LOG.warning("Failed to execute status_bar view %s" % (view,))
-    except:
+    except Exception as e:
```

Comment:
Same as above

Reply:
changed `except Exception as e`: to `except Exception`